### ISSUE TITLE
Include multiple roles for agents

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#868]
See MAB 100 at [HT018857620 (MAB-XML)](http://lobid.org/hbz01/HT018857620). In the json only one role is covered: http://lobid.org/resources/HT018857620.json. Because it's the first time that I saw something like this I got the confirmation by a cataloger that it's possible.

**Comment by ChristophEwertowski** [comment]
Since `role` isn't an array these changes will be an API break. So we have to inform all users before deploying this.

**Comment by acka47** [comment]
In 2016 I asked on the Bibframe list how to record multiple contributions by the same person. Ray Denenberg of LoC [responded](https://listserv.loc.gov/cgi-bin/wa?A2=ind1612&L=BIBFRAME&D=0&P=13768):

> If you want to list two roles, e.g. “illustrator” and “author” for the same person, the rule is, declare two separate Contribution resources.   Two or more role statements within the same Contribution resource should occur only because you want to provide two different representations (e.g. two different URIs) for the same role.

**Comment by ChristophEwertowski** [comment]
Note: Apparently you can `<combine>` like the following in the morph. `Field A + field B` / `field A + field C` etc.

**Comment by ChristophEwertowski** [comment]
The solution which with I came up with can cover up to three roles of an MAB number / entity. To test this I added roles to test resources see `Barenboim, Daniel` at [HT018857620 (production, json)](http://lobid.org/resources/HT018857620.json) / [HT018857620 (test, json)](http://test.lobid.org/resources/HT018857620.json) and `Albert-Ludwigs-Universität Freiburg` at [HT018770176 (test, json)](http://test.lobid.org/resources/HT018770176.json).
I will open an issue at the metamorph repository for using loops. With that it would be possible to use all roles if there are four or more and it would reduce the lines of code dramatically.
